README.md

```diff
@@ -1 +1,50 @@
-# earthly-mode
\ No newline at end of file
+# earthly-mode
+
+Major mode for editing Earthly Earthfile (https://docs.earthly.dev/docs/earthfile)
```

Comment:
```suggestion
Major mode for editing Earthly Earthfile (https://docs.earthly.dev/docs/earthfile) developed by @wingyplus, @TheCoreMan and @duck1123
```

Reply:
Added in https://github.com/earthly/earthly-mode/pull/1/commits/dcbdeb306b5fe9b2fa231b22051d519470cd5f96.